packages/synapse-react-client/package.json

```diff
@@ -228,7 +229,8 @@
     "build:esbuild": "rimraf dist/umd && node esbuild.config.mjs",
     "serve:docs": "pnpm storybook",
     "build:docs": "pnpm build:storybook",
-    "build:storybook": "storybook build && touch storybook-static/.nojekyll",
+    "build:build:gh-pages": "pnpm build-storybook && touch storybook-static/.nojekyll",
+    "build-storybook": "storybook build",
```

Comment:
Chromatic expects a `build-storybook` command. They let us override with a custom command, but that was causing issues for me where the job would fail without a meaningful error message.